src/components/ResourcesCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import InformationCards from "./cards/InformationCards";
+import HeaderCArd from "./cards/HeaderCArd";
+
+interface Props {
+  Entries: any;
+}
+
+const ResourcesCard = ({ Entries }: Props) => {
+  const { title, description, entries, colSpan, rowSpan, heightMultiplier } =
```

Comment:
This should be named `Block` and it's not a card

Reply:
fixed